apps/src/blockly/addons/cdoKeyboardNavigation.ts

```diff
@@ -22,8 +21,6 @@ export function initializeKeyboardNavigation(
   });
 
   enableShortcutModalEscape();
-  // Rerun user theme after Keyboard Experiment bug introduces incorrect theme
```

Comment:
Currently, Blockly's new Keyboard Nav plugin has a bug where the wrong theme is set. We've been correcting this here, but it's no longer needed. Now, all labs will ensure they're using a valid theme (whether that comes from user preferences, browser store, lab code or a safe default) as soon as the workspace is created.

Reply:
FYI @hannahbergam